Dockerfile

```diff
@@ -2,7 +2,7 @@
 # cross-compiled, it cannot have ANY "RUN" commands. All binaries are built
 # using the go toolchain on the host and then copied into the build context by
 # scripts/build_docker.sh.
-FROM alpine:latest
+FROM alpine/git:latest
```

Comment:
It seems `alpine/git` adds `openssh` as well, I'm not sure we need/want to pull in `openssh-server` though. Perhaps we should manually install the packages?

https://github.com/alpine-docker/git/blob/28966a5ebc338f9c64ccd4384f94108d6db1a6c4/Dockerfile#L3-L5

I.e. change `openssh` to `openssh-client`.

Reply:
We can't manually install packages without massively refactoring our build pipeline to add runners for other architectures, so we need to find an image that already does what we need.

openssh-server isn't a big deal to me, but the volume is quite annoying. :/